.circleci/config.yml

```diff
@@ -45,88 +44,38 @@ jobs:
           command: |
             pipenv run flake8 .
       - run:
-          name: Run tests
```

Comment:
What's the purpose of merging these steps? I think it would be good to differentiate between tests failing and code coverage uploading failing.

Reply:
mistake